docs/operators/kafkaInputOperator.md

```diff
@@ -255,28 +262,169 @@ Below is the code snippet:
 @ApplicationAnnotation(name = "KafkaApp")
 public class ExampleKafkaApplication implements StreamingApplication
 {
-@Override
-public void populateDAG(DAG dag, Configuration entries)
-{
-  KafkaSinglePortByteArrayInputOperator input =  dag.addOperator("MessageReader", new KafkaSinglePortByteArrayInputOperator());
+  @Override
+  public void populateDAG(DAG dag, Configuration entries)
+  {
+    KafkaSinglePortByteArrayInputOperator input =  dag.addOperator("MessageReader", new KafkaSinglePortByteArrayInputOperator());
+    ConsoleOutputOperator output = dag.addOperator("Output", new ConsoleOutputOperator());
+    dag.addStream("MessageData", input.outputPort, output.input);
+  }
+}
+```
+Below is the configuration for “test” Kafka topic name and
+“localhost:2181” is the zookeeper forum:
+
+```xml
+<property>
+  <name>dt.operator.MessageReader.prop.topic</name>
+  <value>test</value>
+</property>
 
-  ConsoleOutputOperator output = dag.addOperator("Output", new ConsoleOutputOperator());
+<property>
+  <name>dt.operator.KafkaInputOperator.prop.zookeeper</nam>
+  <value>localhost:2181</value>
+</property>
+```
 
-  dag.addStream("MessageData", input.outputPort, output.input);
-}
+
+### Kafka Input Operator for Kafka 0.9.x
+
+Package: `org.apache.apex.malhar.kafka`
+
+Maven Artifact: [malhar-kafka](https://mvnrepository.com/artifact/org.apache.apex/malhar-kafka)
+
+This version uses the new 0.9 version of consumer API and works with Kafka broker version 0.9 and later.
+The operator is fault-tolerant, scalable and supports input from multiple clusters and multiple topics in a single operator instance.
+
+#### Pre-requisites
+
+This operator requires version 0.9.0 or later of the Kafka Consumer API.
+
+### AbstractKafkaInputOperator
+
+#### Ports
+----------
+
+This abstract class doesn't have any ports.
+
+#### Configuration properties
+----------------------------
+
+-   ***clusters*** - String[]
```

Comment:
This should be formatted as an HTML table to be consistent with the others above.